app/config/config-default.js

```diff
@@ -52,6 +52,12 @@ module.exports = {
     // custom padding (CSS format, i.e.: `top right bottom left`)
     padding: '12px 14px',
 
+    // default window width
+    windowWidth: 500,
```

Comment:
maybe `defaultWindowSize: {width:500, height: 370}` to make it clear that it's default without needing the comment. I don't know what's best 😄 @chabou ?